modules/tinymce/src/core/test/ts/browser/content/insert/NamespaceElementsTest.ts

```diff
@@ -11,12 +11,13 @@ describe('browser.tinymce.core.content.insert.NamespaceElementsTest', () => {
     valid_children: '+p[math]'
   }, [], true);
 
-  it('TINY-10237: Inserting SVG elements but filter out things like scripts', () => {
+  // Updated for TINY-11332: Remove html inside SVG
+  it('TINY-10237: Inserting SVG elements but filter out things like scripts and invalid children', () => {
     const editor = hook.editor();
     editor.setContent('<p>ab</p>');
     TinySelections.setCursor(editor, [ 0, 0 ], 1);
     editor.insertContent('<svg><circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red"><desc><script>alert(1)</script><p>hello</p></circle></a></svg>');
-    TinyAssertions.assertContent(editor, '<p>a<svg><circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red"><desc><p>hello</p></desc></circle></svg>b</p>');
+    TinyAssertions.assertContent(editor, '<p>a<svg><circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red"><desc></desc></circle></svg>b</p>');
```

Comment:
We could still check if text is properly retained in `desc`.

Reply:
Added new test for keeping content